src/site/resources/index.html

```diff
@@ -62,7 +62,7 @@ <h1 class="logo"><img src="img/logo-full.svg" alt="Google Cloud Platform" /></h1
                         <pre>&lt;dependency&gt;
   &lt;groupId&gt;com.google.gcloud&lt;/groupId&gt;
   &lt;artifactId&gt;gcloud-java&lt;/artifactId&gt;
-  &lt;version&gt;0.0.6&lt;/version&gt;
+  &lt;version&gt;SITE_VERSION&lt;/version&gt;
```

Comment:
Is this intended to be a template value? Is there special notation to be used, e.g. `{{SITE_VERSION}}`?